apps/src/code-studio/progress.js

```diff
@@ -27,8 +27,7 @@ progress.renderStageProgress = function (stageData, progressData, scriptName,
     stages: [stageData]
   }, currentLevelId, saveAnswersBeforeNavigation);
 
-  store.dispatch(mergeProgress(_.mapValues(progressData.levels,
-    level => level.submitted ? SUBMITTED_RESULT : level.result)));
+  store.dispatch(mergeProgress(_.mapValues(progressData.levels, level => level.result)));
```

Comment:
They should have level.result - levels that have been submitted and peer reviewed should turn red or green